avo-hq/avo
# feature: customizable resource controls

# Description


Fixes https://github.com/avo-hq/avo/issues/1082
Pre-release: [2.11.3.pre.1](https://rubygems.org/gems/avo/versions/2.11.3.pre.1)



# Checklist:




## Manual review steps


1. Go to a fish
1. Observe the buttons are like the ones in the screenshot below.
![CleanShot 2022-08-09 at 02 28 43](https://user-images.githubusercontent.com/1334409/183531553-2e7727f0-0b1b-4315-8b4b-b46753d99851.jpg)

Manual reviewer: please leave a comment with output from the test if that's the case.

## Screenshots

![CleanShot 2022-07-29 at 14 01 11@2x](https://user-images.githubusercontent.com/1334409/181745869-f3ea4a70-a2f9-452a-b84d-9840accee112.jpg)



https://user-images.githubusercontent.com/1334409/181796880-f7dcf743-e516-4292-8d4c-254220d1b120.mp4
